test/es2015/module-support-assessment-test.mjs

```diff
@@ -0,0 +1,14 @@
+"use strict";
+import referee from "referee";
+import sinon from "../../lib/sinon";
+import * as aModule from "./a-module";
+const {assert} = referee;
+
+describe("Explicit lack of support for stubbing Modules", function() {
+    it("should give a proper error message for modules without default export", function() {
+        assert.exception(function() {
+            sinon.stub(aModule, "anExport").returns(400);
```

Comment:
We will have the same problem for spies, won't we?

Reply:
Yes. Pushed a fix for this.